src/megameklab/com/ui/util/SpringUtilities.java

```diff
@@ -71,7 +64,7 @@ public static void makeGrid(Container parent,
         try {
             layout = (SpringLayout)parent.getLayout();
         } catch (ClassCastException exc) {
-            System.err.println("The first argument to makeGrid must use SpringLayout.");
+            LogManager.getLogger().error(exc);
```

Comment:
You'll need to use error("", exc) instead, as that will log it as an exception instead of as an object (design limitation on the Log4j side of things)

Reply:
Ah ok, I had been wondering about this...
Changed it.